.azure-pipelines/steps/download-artifact.yml

```
@@ -0,0 +1,21 @@
+# .azure-pipelines/steps/download-artifact.yml
+parameters:
+  artifact: ''
+  path: ''
+  patterns: ''
+  displayName: ''
+  condition: ''
+  # Defaults
+  timeoutInMinutes: 45
+  retryCountOnTaskFailure: 3
+
+steps:
+- task: DownloadPipelineArtifact@2
+  displayName: ${{ parameters.displayName }}
```

If we get rid of the `displayName` parameter, we can just display the artifact name here
```suggestion
  displayName: "Download ${{ parameters.artifact }}"
```

Alternatively, you could use a coalesce to make it optional, and delete it in _most_ callsites:

```yaml
  displayName: "${{ coalesce(parameters.DisplayName, 'Download 
 ' + parameters.artifact }}"
```

(I'm not 100% on the syntax of the above, but I _think_ it should work 😅 )

---

yes, removing the displayName would save us several lines of code. Thanks!